aiohttp_jwt/middleware.py

```diff
@@ -70,23 +62,36 @@ def JWTMiddleware(
                     )
 
                 if token is not None:
+                    if not isinstance(token, bytes):
+                        token = token.encode()
+
                     try:
-                        if not isinstance(token, bytes):
-                            token = token.encode()
                         decoded = jwt.decode(
                             token,
                             secret_or_pub_key,
                             algorithms=algorithms,
                         )
-                        request[request_property] = decoded
-                        if store_token and isinstance(store_token, str):
-                            request[store_token] = token
                     except jwt.InvalidTokenError as exc:
                         logger.exception(exc, exc_info=exc)
                         raise aiohttp.web.HTTPForbidden(
                             reason='Invalid authorization token',
                         )
 
+                    request[request_property] = decoded
+
+                    if callable(is_revoked):
+                        if await invoke(partial(
+                            is_revoked,
```

Comment:
Perhaps we could avoid the _decoded_ arg in _is_revoked_ func due to the request already has this information(request[request_property] = decoded).

Reply:
Or perhaps we could place all request modifications below all checks(for clarity)

Reply:
second variant makes sense